examples/flow.cpp

```diff
@@ -150,6 +154,20 @@ try
         std::cout << "**********************************************************************\n\n";
     }
 
+#ifdef _OPENMP
+    if (!getenv("OMP_NUM_THREADS")) {
+        //Default to max(4,#cores) threads,
+        //not number of cores (unless ENV(OMP_NUM_THREADS) is defined)
+        int num_cores = omp_get_num_procs();
+        int num_threads = std::min(4, num_cores);
```

Comment:
The comment says `std::max()`, but the code says `std::min()`.  Which is the intended semantics?